src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourceMappingOptionsControl.xaml

```diff
@@ -0,0 +1,56 @@
+<UserControl x:Class="NuGet.Options.PackageSourceMappingOptionsControl"
```

Comment:
A button has no border on the side. Can we add missing border?

![image](https://user-images.githubusercontent.com/1192347/183227651-152724a2-1567-491e-be96-a613c9389781.png)

Reply:
https://github.com/NuGet/Client.Engineering/issues/1834